src/pages/index.js

```diff
@@ -74,7 +74,7 @@ function PolicyPersistence() {
             In Casbin, the policy storage is implemented as an adapter(aka middleware for Casbin). To keep light-weight, we don't put adapter code in the main library (except the default file adapter). A complete list of Casbin adapters is provided as below. Any 3rd-party contribution on a new adapter is welcomed, please inform us and I will put it in this list:) For details of adapters, please refer to the documentation: <Link to="/docs/adapters">adapters</Link>
           </div>
           <div>
-            <img src="/img/store.png" width="600" height= "660" frameborder="0" scrolling="no" />
+            <img src="/img/store.png" width="600" height= "660"></img>
```

Comment:
`frameborder` and `scrolling`  are not `<img>` 's attribute.

Reply:
OK